attendance/resources/attendance.py

```diff
@@ -10,9 +12,17 @@ def get(self):
 
     def post(self):
         body = request.get_json()
-        attendance = Attendance(**body).save()
-        id = attendance.id
-        return {'id': str(id)}, 200
+        attendance = Attendance(**body)
+        apiResponse = urllib.request.urlopen("http://management/manage/geo-fence?universityName="+attendance.school).read()
```

Comment:
Should read the property from a configuration

Reply:
done